drogon/assist/passwdhash.cpp

```diff
@@ -9,6 +9,7 @@
 static Botan::System_RNG rng;
 using namespace drogon;
 using namespace drassist;
+using namespace std;
```

Comment:
Please add `std::` where needed and remove this line. For example, use std::string_view instead of string_view.

Reply:
Okay, got it. I will do this once the other stuff, is fixed since I am unable to locate all the other places it requires std:: . Sorry for the delay. I went to Universal Studios on the 3rd I live 10 minutes away. And then it was the 4th of July...